### DISCUSSION TITLE
Font of Mastodon logo

### DISCUSSION THREAD

**Discussion by PeterAlfonsLoch** [mastodon/mastodon#20482] [Q&A]
Hello, I would like to follow the branding guidelines. Yet I can't find any infomration to what fonts to use?

**Comment by KrisEnnkay** [comment]
I don't know the official answer, but I found these matches when doing a Font Match search in Adobe.

![image](https://user-images.githubusercontent.com/118054397/201472964-0e8f2b60-77ce-4db9-8bd7-a37eff75d0d1.png)
![image](https://user-images.githubusercontent.com/118054397/201472976-2b89813e-85b2-425a-a176-45cb446a2491.png)

**Reply by KrisEnnkay** [reply]
Unfortunately, all but the All Round Gothic Demi have a sort of serif "m" so none of those match exactly.

**Comment by f2k1de** [comment]
Seems to be Gilroy Bolt (paid font) with a custom m https://discourse.joinmastodon.org/t/mastodon-font-request/1437
